paragraphs.module

```diff
@@ -381,6 +381,15 @@ function paragraphs_menu() {
     'type' => MENU_DEFAULT_LOCAL_TASK,
   );
 
+  $items['admin/structure/paragraphs/%paragraphs_bundle/clone'] = array(
+    'title' => 'Clone Paragraphs type',
```

Comment:
Throughout the code, let's standardize how we write `Paragraphs type` in front-facing text, comments, docblocks, etc. I think this is currently the format in use in the rest of the module (e.g. menu items, etc. although I admit it feels a bit of a mouthful). Other variations I have seen:

- `paragraph type`
- `paragraphs type`
- `paragraphs types`

Reply:
I think I've done in all relevant places.